path/src/eula.ps1

```
@@ -0,0 +1,148 @@
+		Stop-Job $script:FountStatusServerJob -ErrorAction SilentlyContinue
+		Remove-Job $script:FountStatusServerJob -Force -ErrorAction SilentlyContinue
+		$script:FountStatusServerJob = $null
+	}
+	Remove-Item Env:\FOUNT_INSTALL_WAIT -Force -ErrorAction Ignore
+	if ($script:FountEulaAcceptFile) {
+		Remove-Item -LiteralPath $script:FountEulaAcceptFile -Force -ErrorAction Ignore
+		$script:FountEulaAcceptFile = $null
+	}
+}
+
+function script:Ensure-FountConfig {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Ensure-FountConfig' uses an unapproved verb.

(PSUseApprovedVerbs)